src/Framework/MockObject/Matcher/ConsecutiveParameters.php

```diff
@@ -37,8 +37,10 @@ class ConsecutiveParameters extends StatelessInvocation
     private $invocations = [];
 
     /**
-     * @throws \PHPUnit\Framework\Exception
+     * @var array
```

Comment:
I think this we can make this more specific as `ExpectationFailedException[]`.

Reply:
... same as above. The `__construct` doesn't throw any exception. It prefills it's object properties only